src/pages/Room/index.js

```diff
@@ -17,15 +19,13 @@ export default function Room() {
   return (
     <Body>
       <ChannelPage>
-        <Header>
-          <button type="button" onClick={handleClick} className="backBtn">
-            {"<"}
-          </button>
-          <h2>룸을 선택하세요</h2>
-        </Header>
+        <BackBtn type="button" onClick={handleClick}>
+          {"<"}
+        </BackBtn>
+        <Header title="룸을 선택하세요." channelId={id} roomId="none" />
         <WrapChannelUL>
           {RoomArr.map((it) => (
-            <RoomList key={it} id={it} text={`${it} 룸`} />
+            <RoomList key={it} channelId={id} id={it} text={`${it} 룸`} />
```

Comment:
props명을 약어인 it이 아니라 명시적으로 추상화를 해주시면 좋을 것 같습니다.
클린코드 서적에서도 약어는 지양하라고 되어 있고, 이 부분을 한번 보시면 그래도 조금은 도움 되실 것 같아 링크 남겨놓아봅니다.

key값도 혹시 index이라면 unique한 값으로 변경해주시는게 좋을 것 같습니다.

참고자료 :https://kyeoneee.tistory.com/55
https://reactjs.org/docs/lists-and-keys.html

```suggestion
            <RoomList key={it} channelId={id} id={it} text={`${it} 룸`} />
```

Reply:
넵 확인해보겠습니다.